plugins/discord.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	name: 'discord',
+	description: 'Replies with discord link',
+	permissions: [],
+	execute(message, args, user, bot, event, plugins) {
+		var discordLink = "discord.com/test" //CHANGE THIS TO MATCH YOUR DISCORD LINK
+		bot.sendMessage(`Make sure to join our awesome discord: ` + discordLink);
```

Comment:
```suggestion
		bot.sendMessage(`Make sure to join our awesome discord: ${discordLink}`);
```

If you already use Javascript-Template-String ( `` ) then at least use it 👍

Reply:
Can you make the discord link set by a config file.

Reply:
Please update your PR for 2.0.0 new way of doing plugins :X sorry!